### PR TITLE
Predictions: data info displayed in the status bar

### PR DESCRIPTION
**Pull request by aturanjanin**
##### Description of changes
Output data info added to the status bar.

##### Includes
- [X] Code changes
- [X] Tests
- [ ] Documentation

### CODE REVIEW

File: Orange/widgets/evaluate/owpredictions.py [hunk 1]
```diff
@@ -299,23 +300,23 @@ def _update_info(self):
...
+        pred_names = [v.name for v in self.predictors.values()]
         n_predictors = len(self.predictors)
-        if self.data:
-            details = plural("{number} instance{s}", len(self.data))
-        else:
-            details = "No data"
         if n_predictors:
             n_valid = len(self._non_errored_predictors())
-            details += plural("\n{number} model{s}", n_predictors)
+            details += plural("\nModel{s}: {number}", n_predictors)
```
Comment (VesnaT) [inline Orange/widgets/evaluate/owpredictions.py:314]: Could you split data and model info with a horizontal line. Now it looks like the model is part of the data.
I recommend something like:
```
Data:
303 instances, 14 variables
Features: 13 (7 categorical, 6 numeric)
Target: categorical
Metas: —
---------------------------------------
Models:
2 models (Random Forest, Logistic Regression)
```
If there are no models, use `No models on input` to be consistent with the data info.

**Comment by VesnaT** [comment]
Can you pleas rebase to master?

**Comment by aturanjanin** [comment]
@VesnaT Rebased now, thanks.

File: Orange/widgets/evaluate/owpredictions.py [hunk 2]
```diff
@@ -296,30 +297,36 @@ def _set_errors(self):
...
             return
 
-        summary = str(len(self.data)) if self.data else "0"
+        summary = len(self.data) if self.data else "0"
         details = self._get_details()
         self.info.set_input_summary(summary, details)
 
     def _get_details(self):
+        details = format_summary_details(self.data) if self.data else \
+            "No data on input"
```
Comment (VesnaT) [inline Orange/widgets/evaluate/owpredictions.py:311]: This is inconsistent with other widget that accept multiple inputs.
I propose:


Data:
No data on input.
- - -
Model:
No model on input.

File: Orange/widgets/evaluate/owpredictions.py [hunk 3]
```diff
@@ -296,30 +297,36 @@ def _set_errors(self):
...
             self.Warning.wrong_targets.clear()
 
-    def _update_info(self):
+    def _set_input_summary(self):
         if not self.data and not self.predictors:
             self.info.set_input_summary(self.info.NoInput)
             return
 
-        summary = str(len(self.data)) if self.data else "0"
+        summary = len(self.data) if self.data else "0"
```
Comment (VesnaT) [inline Orange/widgets/evaluate/owpredictions.py:305]: `"0"` could be an `int`

**Comment by VesnaT** [comment]
The widget initial state (widget without inputs) is missing the info.

**Comment by janezd** [comment]
@aturanjanin, #4871 has been merged. Could you rebase this PR now, please?

**Comment by aturanjanin** [comment]
@janezd  Rebased now, thanks.

**Comment by PrimozGodec** [comment]
It looks good now for me. Thanks.

**Pull request merged**